File: app/src/main/java/com/ten/lifecat/phone/activity/SettingsActivity.java
package com.ten.lifecat.phone.activity;

import android.support.v7.app.ActionBar;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import com.ten.lifecat.phone.R;

/**
 * TODO 用户设置界面
 */
public class SettingsActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_settings);

        // 导航按钮
        ActionBar actionBar = getSupportActionBar();
        if (actionBar != null) {
            actionBar.setDisplayHomeAsUpEnabled(true);
            actionBar.setHomeAsUpIndicator(R.mipmap.navigation);
        }

        // TODO 用户信息展示与修改
    }
}

File: app/src/main/java/com/ten/lifecat/phone/view/activity/SettingsActivity.kt
package com.ten.lifecat.phone.view.activity

import android.support.v7.app.AppCompatActivity
import android.os.Bundle

import com.ten.lifecat.phone.R

/**
 * TODO 用户设置界面
 */
class SettingsActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_settings)

        // 导航按钮
        val actionBar = supportActionBar
        if (actionBar != null) {
            actionBar.setDisplayHomeAsUpEnabled(true)
            actionBar.setHomeAsUpIndicator(R.mipmap.navigation)
        }

        // TODO 用户信息展示与修改
    }
}
